{"status":"FAIL","msg":"FAIL:\n Uncaught: [object Object]\n    at test262/harness/assert.js:49\n    at anonymous (test262/test/built-ins/Object/prototype/isPrototypeOf/this-value-is-in-prototype-chain-of-arg.js:40)\n\n","path":"test262/test/built-ins/Object/prototype/isPrototypeOf/this-value-is-in-prototype-chain-of-arg.js"}